src/lib/rest/rest.cpp

```diff
@@ -915,21 +917,32 @@ bool urlCheck(ConnectionInfo* ciP, const std::string& url)
 *  NO_VERSION: others (invalid paths)
 *
 */
-static ApiVersion apiVersionGet(const char* path)
+static ApiVersion apiVersionGet(const char* path, const std::string& service, const std::string& subService)
 {
+  metricsMgr.add(service, subService, METRIC_TRANS_IN, 1);
```

Comment:
*the best place would be the one in which the incoming transcation is started, just after getting serv and subserv info from headers*: Well, that is petty much right here where I have put it.

I put it inside that function as the info ngsi9/ngsi10 I don't have elsewhere, not without doing a new investigation of the  URL path, i.e. duplicating the code.

No problem in doing that though, if you prefer to have the functionality separated [ cleaner but slower ]